lib/dkdeploy/php/helpers/http.rb

```diff
@@ -18,20 +18,22 @@ def call_file_on_server(filename, server)
           web_server_port = domain_scheme == 'https' ? 443 : 80
           # Use server configuration, if exists
           web_server_port = server.fetch(:web_server_port) if server.properties.respond_to?(:web_server_port)
+          ssl_verify_mode = server.properties.respond_to?(:ssl_verify_mode) ? server.fetch(:ssl_verify_mode) : OpenSSL::SSL::VERIFY_PEER
 
           url = URI.parse("#{domain_scheme}://#{domain}").merge("/#{filename}")
           url.port = web_server_port
           info "Call URL #{url}"
 
-          http_get_with_redirect url
+          http_get_with_redirect url: url, verify_mode: ssl_verify_mode
         end
 
         # Sends a get request that handles redirects
         #
         # @param url [URI]
         # @param limit [Integer] defines how many redirects are allowed
+        # @param verify_mode [OpenSSL::SSL::verify_mode] ssl verify mode, setting to VERIFY_NONE disables certificate check
         # @return [NET::HTTPResponse]
-        def http_get_with_redirect(url, limit = 5)
+        def http_get_with_redirect(url:, limit: 5, verify_mode: OpenSSL::SSL::VERIFY_PEER)
```

Comment:
Is this a breaking change?
If yes, we need to document this breaking change.
Or can we use http_get_with_redirect(url, limit = 5, verify_mode: OpenSSL::SSL::VERIFY_PEER)?